lms/templates/courseware/accordion.html

```diff
@@ -19,15 +19,15 @@
 <a href="#${chapter['display_id']}-child" role="button" class="button-chapter chapter ${active_class}" id="${chapter['display_id']}-parent" aria-controls="${chapter['display_id']}-child" aria-expanded="false">
     <span class="group-heading ${active_class}" aria-label="${aria_label}">
         <span class="icon fa fa-caret-right" aria-hidden="true"></span>
-        ${chapter['display_name']}
+        ${HTML(chapter['display_name'])}
```

Comment:
@attiyaIshaque Why not do this on Line#13 with Text and HTML escaping . This would escape `aria_label` as well.